audit/auditLog.json

```diff
@@ -116,7 +116,7 @@
       "auditCompletedOn": "10.01.2025",
       "auditedBy": "Cantina (security firm)",
       "auditorGitHandle": "cantinaxyz",
-      "auditReportPath": "./audit/reports/report-cantinacode-lifi-1216.pdf",
+      "auditReportPath": "./audit/reports/2025.01.10_Cantina Pre-Comp Audit.pdf",
```

Comment:
I recommend renaming file so it doesn't contain any dashes or "Audit" suffix as it is already in audit folder. The exception is _ReAudit suffix when we get reaudit.

I would rename it to: `2025.01.10_CantinaPreComp.pdf`.

btw: Do we have any guidelines that describe naming conventions for audit file names?

Reply:
good point. Updated

Reply:
got it! thanks!